tests/app/main/views/test_index.py

```diff
@@ -188,12 +188,12 @@ def test_message_status_page_contains_message_status_ids(client_request):
     assert page.find(id='text-message-statuses')
 
 
-def test_message_status_page_contains_link_to_support(client_request):
-    page = client_request.get('main.message_status')
-    sms_status_table = page.find(id='text-message-statuses').findNext('tbody')
+# def test_message_status_page_contains_link_to_support(client_request):
```

Comment:
It looks like this link should still be there. If this test is failing, did the element move to a different spot on the page?